usaspending_api/disaster/tests/fixtures/helpers.py

```diff
@@ -36,8 +36,10 @@ def post_for_spending_endpoint(client, url, **kwargs):
         return resp
 
     @staticmethod
-    def post_for_count_endpoint(client, url, def_codes=None):
-        if def_codes:
+    def post_for_count_endpoint(client, url, def_codes=None, award_type_codes=None):
+        if award_type_codes:
+            request_body = json.dumps({"filter": {"def_codes": def_codes, "award_type_codes": award_type_codes}})
+        elif def_codes:
```

Comment:
Why check for def_codes if the above line operates as if they are always present?

Perhaps use similar conditionals as `post_for_amount_endpoint()`

Reply:
They are not always present, as can be seen in https://github.com/fedspendingtransparency/usaspending-api/pull/2569/files#diff-a913bf1e1f3808c8731f33b728e68d3eR89 as well as a few other places.

Reply:
I could have rephrased the question, it's currently a non-issue since there aren't tests for when `award_type_codes` is present but `def_codes` is missing. A future dev can duplicate the conditional in `post_for_amount_endpoint()` to handle both if that test case is introduced.